src/main/java/io/jenkins/plugins/artifact_manager_jclouds/JCloudsArtifactManager.java

```diff
@@ -132,7 +132,10 @@ public void archive(FilePath workspace, Launcher launcher, BuildListener listene
             String blobPath = getBlobPath(path);
             Blob blob = blobStore.blobBuilder(blobPath).build();
             blob.getMetadata().setContainer(provider.getContainer());
-            blob.getMetadata().getContentMetadata().setContentType(contentTypes.get(entry.getKey()));
+            String contentType = contentTypes.get(entry.getKey());
```

Comment:
```suggestion
            String contentType = contentTypes.get(entry.getValue());
```
 if I follow correctly?

Reply:
Yes I think so; applied in https://github.com/jenkinsci/artifact-manager-s3-plugin/pull/349/commits/56b1cb7237d04a3bcc197ff3657be43ceeb7e44f.